frontend/src/main/webapp/index.html

```diff
@@ -6,6 +6,8 @@
   </head>
   <body>
     <h1>Trip Me Out! Fill me in more like!</h1>
+    <a href="pagevisit.html">Page Visit</a>
+    <a href="triplist.html">Trip List</a>
```

Comment:
not sure why this is here...

Reply:
After I fixed the merge conflict I decided just to make both of the index.html I was working on the same